### PR TITLE
Finished basic schema

### PR DESCRIPTION
**Pull request by dividedsky**
database on heroku is updated, as well

### CODE REVIEW

File: knexfile.js [hunk 1]
```diff
@@ -2,13 +2,12 @@
 require('dotenv').config();
 const pg = require('pg');
 
-pg.defaults.ssl = false; // this needs to be false in development, true when using heroku db
+pg.defaults.ssl = true; // this needs to be false in development, true when using heroku db
```
Comment (timh1203) [inline knexfile.js]: this comment helps a lot!

File: server/data/migrations/20190213212405_followups.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('followups', tbl => {
```
Comment (timh1203) [inline server/data/migrations/20190213212405_followups.js]: Are we using "followup" for internal purposes and "refreshers" for front end?

Comment (dividedsky) [reply server/data/migrations/20190213212405_followups.js]: Good point, I was maybe on autopilot, but perhaps this should be renamed to refreshers

Comment (kelfro) [reply server/data/migrations/20190213212405_followups.js]: Update this to refreshers and I'll go ahead and merge.

**Pull request merged**